.ci/Jenkinsfile

```diff
@@ -74,9 +74,11 @@ pipeline {
         tag pattern: '(v)?\\d+\\.\\d+\\.\\d+', comparator: 'REGEXP'
       }
       steps {
-        withCredentials([string(credentialsId: "${GITHUB_TOKEN_CREDENTIALS}", variable: 'GITHUB_TOKEN')]) {
-          dir("${BASE_DIR}") {
-            sh 'curl -sL https://git.io/goreleaser | bash'
+        withMageEnv(){
```

Comment:
We should also set `GO_VERSION` environment variable, or add a `.go-version` file to pin the version of Go, if not we depend on the [default in the library](https://github.com/elastic/apm-pipeline-library/blob/683617e72b45b1fec13e9ac0f92fc059fc7d417b/vars/goDefaultVersion.groovy#L41).

Or we can also set a version here, as `withMageEnv(version: ...)`.

Reply:
Thanks for suggestion, added .go-version file.